oxd-server/src/test/java/org/gluu/oxd/server/UpdateSiteTest.java

```diff
@@ -24,10 +24,10 @@
  */
 public class UpdateSiteTest {
 
-    @Parameters({"host", "opHost", "redirectUrls"})
+    @Parameters({"host", "opHost", "redirectUrls", "isLoadTest"})
```

Comment:
redundant, we should not mess normal and load tests

Reply:
Done